tools/specs/Microsoft.SqlServer.Server.nuspec

```
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata minClientVersion="2.12">
+    <id>Microsoft.SqlServer.Server</id>
+    <version>1.0.0</version>
+    <title>Microsoft.SqlServer.Server</title>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <projectUrl>https://aka.ms/sqlclientproject</projectUrl>
+    <icon>dotnet.png</icon>
+    <repository type="git" url="https://github.com/dotnet/sqlclient" />
+    <description>This is the helper library for Microsoft.Data.SqlClient for cross framework compatibility support of UDT types.
+    
+Available Types:
+Microsoft.SqlServer.Server.IBinarySerializer
+Microsoft.SqlServer.Server.InvalidUdtException
+Microsoft.SqlServer.Server.SqlFacetAttribute
+Microsoft.SqlServer.Server.SqlFunctionAttribute
+Microsoft.SqlServer.Server.SqlMethodAttribute
+Microsoft.SqlServer.Server.SqlUserDefinedAggregateAttribute
+Microsoft.SqlServer.Server.SqlUserDefinedTypeAttribute
+Microsoft.SqlServer.Server.DataAccessKind
+Microsoft.SqlServer.Server.SystemDataAccessKind
+Microsoft.SqlServer.Server.Format
+
+</description>
+    <releaseNotes>https://go.microsoft.com/fwlink/?linkid=2090501</releaseNotes>
```

I created a new fwlink for this package so that we can control it separately from MDS.
```suggestion
    <releaseNotes>https://go.microsoft.com/fwlink/?linkid=2185441</releaseNotes>
```